homework-5.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+//First part homework
+
+function durationBetweenDates(startDate = '01 Jan 2000',
+                              endDate = '02 Jan 2000',
+                              unit = 'days') {
+
+    const start = new Date(startDate);
+    const end = new Date(endDate);
+
+    const difference = Math.abs(end - start);
+
+    let multiplier;
+    if (unit === 'seconds') multiplier = 1000;
```

Comment:
```suggestion
    if (unit === 'seconds') {
      multiplier = 1000;
    }
```

Відформатуйте код належним чином всюди)